client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.ts

```diff
@@ -233,7 +246,7 @@ export class ParticipantDetailViewComponent extends BaseMeetingComponent {
 
         // TODO: Open groups subscription
         this.groups = this.groupRepo.getViewModelListWithoutSystemGroupsObservable();
-
+        this.updateEditable();
```

Comment:
This should not be called when creating a new user. Also that method does not keep track on route parameter changes.

Reply:
Moved into load user by id